server/limit_rate_gh.go

```diff
@@ -12,9 +12,7 @@ import (
 
 func (s *Server) CheckLimitRateAndSleep() {
 	mlog.Info("Checking the rate limit on Github and will sleep if need...")
-
```

Comment:
Nit: Why is this line getting dropped?

Reply:
The line didn't make the code any more readable in my opinion.